src/db.rs

```diff
@@ -485,6 +492,13 @@ pub struct Builder {
 }
 
 impl Builder {
+    /// Construct a new [Builder] with sensible defaults.
+    ///
+    /// ## Defaults
+    ///
+    /// - `page_size`: 102400 Kib = 100 * 1024 * 1024 bytes.
```

Comment:
I think you meant write_cache_size that's what defaults to 100MiB, not the page size

Reply:
Oh that's right I just confused both while rewriting something and mixed it.

Fixed it and took your suggestion to write it as Mio/Gib.